client/components/AddReview.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { renderAddReview } from '../redux/reviews/reviewActions';
+
+class AddReview extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      comment: '',
+      stars: 5,
+      albumId: this.props.id
+    }
+    this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
we have the babel plugin to allow arrow functions, please refactor those into arrow functions and omit these bind expressions